README.md

```diff
@@ -19,7 +19,7 @@ Conceptlists in Concepticon:
 ## Statistics
 
 
-[![Build Status](https://travis-ci.org/lexibank/sagartst.svg?branch=master)](https://travis-ci.org/lexibank/sagartst)
+[![Build Status](https://travis-ci.org/digling/sinotibetan-data.svg?branch=master)](https://travis-ci.org/digling/sinotibetan-data)
```

Comment:
You should update the remotes of your repository clone - or maybe just re-clone from the new location.